scripts/apps-mock.py

```
@@ -0,0 +1,87 @@
+        print(f'[mock {MODE}] {method} {parsed.path}  query={dict(parse_qs(parsed.query))}  body_bytes={len(body)}  -> action={action}', file=sys.stderr)
+        if action == 'publish' or VERBOSE:
+            auth = self.headers.get('Authorization', '<missing>')
+            print(f'[mock]   Authorization: {auth[:40]}...', file=sys.stderr)
+            ct = self.headers.get('Content-Type', '')
+            if 'multipart' in ct:
+                boundary = ct.split('boundary=')[-1]
+                m = re.search(rb'name="([^"]+)"', body)
+                print(f'[mock]   multipart boundary={boundary[:20]}...', file=sys.stderr)
+                print(f'[mock]   first field name = {m.group(1).decode() if m else "<none>"}', file=sys.stderr)
+        if action is None:
+            self.send_response(404); self.end_headers(); return
```

Ruff (0.15.13)

[error] Multiple statements on one line (semicolon)

(E702)